h5pyd/_hl/group.py

```diff
@@ -295,14 +289,16 @@ def getObjByUuid(collection_type, uuid):
             tgt = getObjByUuid(soft_json['collection'], soft_json['id'])
 
         elif link_class == 'H5L_TYPE_EXTERNAL':
-            # try to get a handle to the file
-            raise IOError("Not implemented")           
-            
+            # try to get a handle to the file and return the linked object...
```

Comment:
This is the relevant part. The rest of changes are due to removal of extraneous whitespace.